style.css

```diff
@@ -0,0 +1,69 @@
+h1,h2{
+    color: rgb(167, 101, 221);
+}
+body {
+    background: linear-gradient(to bottom, #000000 0%,#17cbeb 100%);
+}
+.box {
+    color: rgb(167, 101, 221);
+    width: 375px; 
+    height: 200px; 
+    font-size: 65px;
+    text-align: center; 
+    margin-left: auto;
+    background-color: rgb(12, 7, 1);       
+}
+div {
+    border-radius: 100%; 
+    height: 200px; 
+    width: 200px;  
+}
+.click{
+    color: rgb(167, 101, 221);
+    background-color: rgb(3, 14, 15);
+    height: 100px;
+    width: 100px;
+    font-size: medium;
+    box-align: center;
+}
+.circle1 {
+   background-color: rgb(148, 90, 223); 
+   left: 500px;
+   top: 125px;
+   position: absolute;
+}
+.circle2,.circle3,.circle4,.circle5 { 
+    left: 65px;
+    top: 100px;   
+    position: absolute;
```

Comment:
Use a shared class for this.

The idea is to remove repeated code and to use id's to represent unique elements and classes to group those elements. Id rules will override class rules because they have a higher "specificity". Use that to your advantage to writer cleaner CSS.